content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -739,6 +740,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `UnknownVersionInteroperabilityProxy`: Proxy resource requests to the correct peer kube-apiserver when
   multiple kube-apiservers exist at varied versions.
   See [Mixed version proxy](/docs/concepts/architecture/mixed-version-proxy/) for more information.
+- `UserNamespacesPodSecurityStandards`: Enable Pod Security Standards relaxation for User
+  NamsNamespaces. See [User Namespaces](/docs/concepts/workloads/pods/user-namespaces/#integration-into-pod-security-standards)
+  for more details.
```

Comment:
```suggestion
- `UserNamespacesPodSecurityStandards`: Change enforcement (Pod security admission)
  of pod security standards, for Pods that use user namespaces. See
  [User Namespaces](/docs/concepts/workloads/pods/user-namespaces/#integration-into-pod-security-standards)
  for more details.
```

Reply:
We are not changing the enforcement in any way, we are changing the policies. I think the original text is clearer. Maybe "Enable Pod Security Standards policies relaxation", is better? I changed it to that :)

I've also adjusted the link for the new title you suggested.